wagtail/wagtailembeds/oembed_providers.py

```diff
@@ -301,6 +301,9 @@
     "http://issuu.com/oembed": [
         "^http(?:s)?://(?:www\\.)?issuu\\.com/[^#?/]+/docs/.+$"
     ],
+    "https://www.tumblr.com/oembed/1.0": [
+        "^http(?:s)?://.+.tumblr.com/post/.+$",
```

Comment:
> You must escape the explicit dots in the domain name.

Ah, right. Thanks! I've forgotten to do that. But dots should be escaped like `\\.`, because it's just a string. We compile it later in the `compile_endpoints` function. 

> You can remove the non-capturing group for 's' of 'https'. It could of course be removed on all other oembed regexps.

I'll leave it as is to for now to match other expressions.

> Maybe a non-greedy expression for the subdomain would be better.

We can't use `//[^\.]+` for subdomains because `http://www.something.tumblr.com/post/something_else` will not work.

I'll replace with `.+?`